src/BundledBindings/AdditionalInformation.cs

```diff
@@ -16,29 +16,23 @@ public class AdditionalInformation : IInputBinding
 
         public override BindingInformation? ExtractBinding(AttributeAst attribute, ParameterAst parameter)
         {
-            string? bindingName = WorkerIndexingHelper.GetPositionalArgumentStringValue(attribute, 0);
-            string? name = WorkerIndexingHelper.GetPositionalArgumentStringValue(attribute, 1);
-            object? value = null;
-            if (attribute.PositionalArguments.Count() >= 2 && attribute.PositionalArguments[2].GetType() == typeof(StringConstantExpressionAst))
-            {
-                value = WorkerIndexingHelper.GetPositionalArgumentStringValue(attribute, 2);
-            }
-            else if (attribute.PositionalArguments.Count() >= 2)
-            {
-                value = WorkerIndexingHelper.ExtractOneOrMore(attribute.PositionalArguments[2]);
-            }
+            string bindingName = WorkerIndexingHelper.GetNamedArgumentStringValue(attribute, "BindingName");
```

Comment:
Should we make this `"BindingName"` constants?

Reply:
Done